src/OpenTelemetry.Exporter.Geneva/README.md

```diff
@@ -157,6 +157,58 @@ values.
   * `ILogger<MyPartner.Product.Thing>`: This is marked as pass-through ("*") so
     it will be sanitized as "MyPartnerProductThing" table name
 
+#### How to configure GenevaExporterOptions using dependency injection
+
+* Tracing:
+
+   > [!NOTE]
```

Comment:
This doesn't seem to be rendered correctly, probably due to indentation or format issue:

![image](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/assets/17327289/ff2cef47-ca63-4dc8-b2d3-ad216f6d7505)

Reply:
Took a few tries but I got it to work 🤣 

![image](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/assets/28367120/ab903029-a768-4267-9317-c5b7baf4f4b7)

Looks like you can't nest these things anymore (not even in lists). Lots of folks complaining about that on the discussion.